doc/documentation-main.html

```diff
@@ -1622,8 +1622,60 @@ <h3 class="sourcetitle">Test1.java</h3>
 will make both <tt>test1()</tt> and <tt>test2()</tt> test methods but on top of that, <tt>test2()</tt> now belongs to the group "g1".
 <p>
 
+<!-------------------------------------
+  IGNORE TESTS
+  ------------------------------------>
+
+<h4><a class="section" indent=".." name="ignore-tests">Ignoring tests</a></h4>
 
+TestNG now lets you ignore tests in a new way. You can ignore all the <tt>@Test</tt> methods :
```

Comment:
> TestNG lets you ignore all the <tt>@Test</tt> methods:

(no need for the first sentence and for "now" in a documentation)

Reply:
fixed.